components/AppBar.vue

```diff
@@ -1,38 +1,42 @@
 <template>
-  <v-app-bar app flat clipped-left class="white">
+  <v-app-bar app flat clipped-left class="white" height="120px">
     <v-container>
       <v-row>
-        <nuxt-link to="/">
-          <img src="../assets/logo/logo_text_blue.svg" alt="Covid Watch">
-        </nuxt-link>
-
+        <v-col>
+          <nuxt-link to="/">
+            <img
+              src="../assets/logo/logo_text_blue.svg"
+              alt="Covid Watch"
+              id="logo-desktop"
+            />
+          </nuxt-link>
+        </v-col>
 
         <v-spacer />
         <!--Desktop menu-->
-        <v-toolbar-items class="d-none d-md-flex">
+        <v-toolbar-items class="hidden-sm-and-down">
```

Comment:
Looks like you can write these classes to show/hide at different viewport sizes in much friendlier ways!